electron/native/nvidia-cuda-compositor/render-tahoe-cursor-atlas.cjs

```
@@ -0,0 +1,144 @@
+const { app, BrowserWindow, ipcMain } = require("electron");
+const fs = require("node:fs");
```

Biome (2.4.14)

[error] This variable fs is unused.

(lint/correctness/noUnusedVariables)